build_tools/azure/test_script.sh

```diff
@@ -48,24 +48,8 @@ if [[ "$COVERAGE" == "true" ]]; then
     TEST_CMD="$TEST_CMD --cov-config='$COVERAGE_PROCESS_START' --cov sklearn --cov-report="
 fi
 
-if [[ -n "$CHECK_WARNINGS" ]]; then
-    TEST_CMD="$TEST_CMD -Werror::DeprecationWarning -Werror::FutureWarning -Werror::sklearn.utils.fixes.VisibleDeprecationWarning"
-
-    # Ignore pkg_resources deprecation warnings triggered by pyamg
-    TEST_CMD="$TEST_CMD -W 'ignore:pkg_resources is deprecated as an API:DeprecationWarning'"
-    TEST_CMD="$TEST_CMD -W 'ignore:Deprecated call to \`pkg_resources:DeprecationWarning'"
-
-    # pytest-cov issue https://github.com/pytest-dev/pytest-cov/issues/557 not
-    # fixed although it has been closed. https://github.com/pytest-dev/pytest-cov/pull/623
-    # would probably fix it.
-    TEST_CMD="$TEST_CMD -W 'ignore:The --rsyncdir command line argument and rsyncdirs config variable are deprecated.:DeprecationWarning'"
-
-    # In some case, exceptions are raised (by bug) in tests, and captured by pytest,
-    # but not raised again. This is for instance the case when Cython directives are
-    # activated: IndexErrors (which aren't fatal) are raised on out-of-bound accesses.
-    # In those cases, pytest instead raises pytest.PytestUnraisableExceptionWarnings,
-    # which we must treat as errors on the CI.
-    TEST_CMD="$TEST_CMD -Werror::pytest.PytestUnraisableExceptionWarning"
+if [[ -n "$IGNORE_WARNINGS" ]]; then
+    TEST_CMD="$TEST_CMD -Wignore"
```

Comment:
```suggestion
    # overwrite the filtering defined in `setup.cfg`
    TEST_CMD="$TEST_CMD -Wignore"
```

Reply:
We should not deactivate everything apparently :)

Reply:
instead I did `-Wwarn`, which should work too.